modules/modules.nix

```diff
@@ -118,6 +118,7 @@ let
     (loadModule ./services/window-managers/awesome.nix { })
     (loadModule ./services/window-managers/i3.nix { })
     (loadModule ./services/window-managers/xmonad.nix { })
+    (loadModule ./services/xcape.nix { })
```

Comment:
Since this module uses systemd it should contain `condition = hostPlatform.isLinux`.